src/btree/bt_import.c

```diff
@@ -138,7 +143,7 @@ __wt_import_repair(WT_SESSION_IMPL *session, const char *uri, char **configp)
     F_SET(ckpt, WT_CKPT_UPDATE);
     WT_ERR(__wt_buf_set(session, &ckpt->raw, checkpoint->data, checkpoint->size));
     WT_ERR(__wt_meta_ckptlist_update_config(session, ckptbase, config_tmp, &config));
-    __wt_verbose(session, WT_VERB_CHECKPOINT, "import metadata: %s", config);
+    __wt_verbose(session, WT_VERB_BACKUP, "import metadata: %s", config);
```

Comment:
Instead of two separate identical lines, you can make it `WT_VERB_BACKUP | WT_VERB_CHECKPOINT`.

Reply:
removed